src/IRVisualization.h

```diff
@@ -0,0 +1,189 @@
+#ifndef IRVISUALIZATION_H
+#define IRVISUALIZATION_H
+
+#include "FindStmtCost.h"
+#include "IRVisitor.h"
+
+#include <set>
+#include <unordered_map>
+
+using namespace std;
+using namespace Halide;
+using namespace Internal;
+
+#define MAX_CONDITION_LENGTH 35
```

Comment:
Is it possible to use ```constexpr``` for these?

Reply:
And if these do not need to be exported, possibly move them into the .cpp file where they are used.